### PR TITLE
Added a project and updated bio

### PR DESCRIPTION
**Pull request by Sakshamkapur**
Hi! I have added a project and updated my bio and title :)

### CODE REVIEW

File: content/participants/ttjc@Sakshamkapur/index.md [hunk 1]
```diff
@@ -1,55 +1,53 @@
...
 
-Hi! I am professional Frontend Developer with about an year of experience in reactjs and node.
+Hi! I am a professional Frontend Developer with about 1.5 years of experience in reactjs and node.
 
-### introduction
+### Introduction
 
-I am confident with JavaScript , experienced with ReactJS,ReactNative,NodeJS
-Always ready to develop something great. :)
+I am confident with my skills in JavaScript and love to write js code and read blog about intresting features in frameworks. I have about 1.5 yr of professional experience in product based startup mainly in reactjs and reactnative.
```
Comment (khusharth) [inline content/participants/ttjc@Sakshamkapur/index.md:16]: * interesting*

File: content/participants/ttjc@Sakshamkapur/index.md [hunk 2]
```diff
@@ -70,3 +68,15 @@ _stack_ built in html,css,javascript.
 _hosted link_  https://sakshamkapur.github.io/2048/2048.html
 
 _github link_ https://github.com/Sakshamkapur/2048
+
+#### Blogit: A Bloging Platform
```
Comment (khusharth) [inline content/participants/ttjc@Sakshamkapur/index.md:72]: Hey! @Sakshamkapur Only projects done during the challenge are accepted for the challenge. As this was done 2 years ago please remove this project.

**Comment by Sakshamkapur** [comment]
Hi! I have resolved the changes required also wanted to ask if i can add changes to older repo and then submit?

**Comment by khusharth** [comment]
I didn't got your question. What do you mean by older repo here?

Also you have made changes inside  **package-lock.json** file. Only changes inside your index.md files are allowed please revert that.

**Pull request closed without merging**